docs/reporting/automating-report-generation.asciidoc

```diff
@@ -4,34 +4,41 @@
 You can automatically generate reports with {watcher}, or by submitting
 HTTP POST requests from a script.
 
-To automatically generate reports with a watch, you need to configure
-{watcher} to trust the {kib} server’s certificate. For more information,
-see <<securing-reporting, Securing Reporting>>.
-
 include::report-intervals.asciidoc[]
 
-To get the URL for triggering a report generation during a given time period:
+[float]
+=== Getting the POST URL
+
+Generating a report either through Watcher or a script requires capturing the **POST
+URL**, which is the URL to queue a report for generation.
+
+To get the URL for triggering PDF report generation during a given time period:
 
-. Load the saved object.
+. Load the saved object in the Visualize editor, or load a Dashboard.
 . Use the timepicker to specify a relative or absolute time period.
 . Click *Share* in the Kibana toolbar.
 . Select *PDF Reports*.
 . Click **Copy POST URL**.
 
-NOTE: The response from this request with be JSON, and will contain a `path` property
-with a URL to use to download the generated report. When requesting that path,
-you will get a 503 response if it's not completed yet. In this case, retry after the
-number of seconds in the `Retry-After` header in the response until the PDF is returned.
+To get the URL for triggering CSV report generation during a given time period:
+
+. Load a saved search in Discover
+. Use the timepicker to specify a relative or absolute time period.
+. Click *Share* in the Kibana toolbar.
+. Select *CSV Reports*.
+. Click **Copy POST URL**.
 
-To configure a watch to email reports, you use the `reporting` attachment type
-in an `email` action. For more information, see
-{stack-ov}/actions-email.html#configuring-email[Configuring Email Accounts].
+[float]
+=== Using Watcher
 
 include::watch-example.asciidoc[]
 
-For more information about configuring watches, see
-{stack-ov}/how-watcher-works.html[How Watcher Works].
+[float]
+=== Using a Script
+
+include::script-example.asciidoc[]
 
+[float]
```

Comment:
The lack of this `[float]` caused this section to be its own page, which seemed wrong